scripts/check.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
 # We need to add the current directory as a safe git directory before running any commands
-git config --global --add safe.directory "$(pwd)"
+git config --global --get-all safe.directory | grep "$(pwd)" > /dev/null
+safedir=$?
+if [[ $safedir -ne 0 ]]; then
```

Comment:
Introducing the `$safedir` variable is not necessary, you can check on `$?` directly or if you want to make it even more compact this should work too:

```
if ! git config --global --get-all safe.directory | grep "$(pwd)" > /dev/null; then
    git config --global --add safe.directory "$(pwd)"
fi
```

Reply:
I was sure I tried this at the time but was getting weird results with this command, I'll try again.

Reply:
Huh that works now, I think I may have wrapped it accidentally with `$()` before for some reason.